_posts/chausuble/2017-05-22-erin-turtlehack.md

```diff
@@ -0,0 +1,29 @@
+--- 
+layout: post
+author: chausuble
+title: "Erin's Turtlehack"
+---
+
+I thought about this before going to bed one night, and I worked on this over the weekend. 
+I wanted to tell a little story with the program - I decided, if you can’t erase a character, 
+just have them caught up in an explosion. So, I decided to make a combat scene. I was just happy to have learned my lesson
+in my basic turtle drawing to draw the background first, then draw the characters on top of it. Otherwise, the background is
+drawn over the characters.
+
+I knew it would take a lot of work - knowing that made it not very frustrating, just a lot of busywork because 
+I don’t have a mind for geometry or angles, so I had to experiment with how .left and .right worked (because they are so fussy to me). 
```

Comment:
Persistence in action.  Well done!  You might find `.seth()` useful in the future for going to a specific heading.